core/src/test/java/org/elasticsearch/search/SearchModuleTests.java

```diff
@@ -99,16 +99,16 @@ public void testRegisteredQueries() throws IOException {
         IndicesQueriesRegistry indicesQueriesRegistry = module.getQueryParserRegistry();
         XContentParser dummyParser = XContentHelper.createParser(new BytesArray("{}"));
         for (String queryName : supportedQueries) {
-            indicesQueriesRegistry.lookup(queryName, dummyParser, ParseFieldMatcher.EMPTY);
+            indicesQueriesRegistry.lookup(queryName, ParseFieldMatcher.EMPTY, dummyParser::getTokenLocation);
```

Comment:
mmm no it does not. If it called the method all the time, what would supplier be for? The `getTokenLocation` is only called when `supplier.get()` is called, at least that is what I see on my machine :)